ex3/server/Command.h

```diff
@@ -9,7 +9,9 @@ class Command {
     std::string description;
     DefaultIO *io;
 public:
-    virtual std::string getDescription() = 0;
+    virtual std::string getDescription();
+
+    virtual DefaultIO *getDefaultIO();
```

Comment:
Why it's not abstract?
```suggestion
    virtual std::string getDescription() = 0;

    virtual DefaultIO *getDefaultIO() = 0;
```

Reply:
because it is implemented